pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
```

Comment:
Yeah, I was aware of this and thought that "ensure state" would be clear enough.  I agree, this can be improved.  What about going declarative, "state_need_multipathd_running"?